src/main/java/com/marginallyclever/makelangelo/Makelangelo.java

```diff
@@ -553,6 +571,8 @@ private JMenu createFileMenu() {
 				WindowEvent windowClosing = new WindowEvent(mainFrame, WindowEvent.WINDOW_CLOSING);
 				Toolkit.getDefaultToolkit().getSystemEventQueue().postEvent(windowClosing);
 			});
+			buttonExit.setAccelerator(KeyStroke.getKeyStroke("ctrl Q"));
+			buttonExit.setMnemonic('q');
```

Comment:
this one is too easy to hit by mistake

Reply:
the mnemonic 'q' (only active if in the File menu (in used to do alt + f , q to quit some application ) ) 
or the Accelerator "ctrl Q" (all the time active)
or both ?

Maybe no mnemonic and just eventualy an accelerator "alt F4" (as this is windows default to quit, just so the user have a reminder ) ?

Reply:
My bad, I miss understood, leave it as you have first done !